warehouse/docker-compose.test.yml

```diff
@@ -16,11 +16,13 @@ services:
       retries: 25
   wh-transformer:
     container_name: wh-transformer
-    image: rudderlabs/rudder-transformer:latest
+    image: rudderlabs/develop-rudder-transformer:master
```

Comment:
Any specific reason why we are using develop image?

Reply:
I wanted to ensure we use the same image the main docker-compose.yaml is using